packages/components/src/local/molecules/chat-message/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+/* Import Types */
+import Props from './types/props'
+
+/* Import Stylesheet */
+import styles from './styles.module.scss'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faUserSecret } from '@fortawesome/free-solid-svg-icons'
+import Box from '@material-ui/core/Box'
+import moment from 'moment'
+import Badge from '../../atoms/badge'
+import Paragraph from '../../atoms/paragraph'
+
+const useStyles = makeStyles({
```

Comment:
This is personal preference @sanjayradadiya - but could you put the styles in `styles.module.scss` rather than using the `makeStyle` call please?   We're not making heavy use of material, so I'm trying to reduce its usage.

Reply:
okay

Reply:
but how can i make color dynamic if i am adding into styles.module.scss ?

Reply:
This is how we do it for non-chat messages:
![image](https://user-images.githubusercontent.com/1108513/112306063-504b6200-8c97-11eb-848f-072b3f9ae1ea.png)

Reply:
Yes, Got it